packages/schemas/tables/applications.sql

```diff
@@ -23,3 +23,8 @@ create index applications__id
 
 create index applications__is_third_party
   on applications (tenant_id, is_third_party);
+
+create unique index applications__protected_app_metadata_host
```

Comment:
just curious what's the difference between a unique index and a unique constraint?

Reply:
Index will improve performance (and use more space)